RELEASES.md

```diff
@@ -22,172 +26,230 @@ Below are the versioning schemes used for publishing debs:
+        * This will generate a list of commits that could be included in the changelog.
       * You don't need to include all of the commits generated. Remember that the changelog should
         be read by the user to understand the new features/modifications in the package. If you
         think a commit will not add that much to the user experience, you can drop it from the
         changelog
-      * To structure the changelog you can use the other entries as example. But we basically try
-        keep this order: debian changes, new features/modifications, testing
- 3. Start building the package:
-    * *WARNING* Build the package in a clean environment. The reason for that is because the package
-      will contain everything that it is present in the folder. If you are storing credentials or
-      other sensible development information in your folder, they will be uploaded too when we send
-      the package to the ppa. A clean environment is the safest way to perform this.
-    * Build the necessary artifacts that allow building the package
-      * Guarantee that you have a gpg key in the system. You will use that gpg key to sign the
-        package.
-        * If you don't yet have a gpg key set up, follow the instructions
-          [here](https://help.launchpad.net/YourAccount/ImportingYourPGPKey) to create a key,
-          publish it to `hkp://keyserver.ubuntu.com`, and import it into Launchpad.
-      * We can achieve that by running the `build-package` command when your current folder is the
-        top of source tree. This script is also found on the [uss-tableflip](https://github.com/canonical/uss-tableflip) repo.
+      * To structure the changelog you can use the other entries as example. But we basically try to
+        keep this order: debian changes, new features/modifications, testing. Within each section, bullet points should be alphabetized.
+        
+    c. Create a PR on github into the release branch. Ask in the UA channel on mattermost for review.
+
+2. After the release PR is merged, tag the head of the release branch with the version number, e.g. `27.1`. Push this tag to Github.
+
+3. Build the package for all Ubuntu releases and upload to `ppa:ua-client/staging`
+
+    a. Clone the repository in a clean directory and switch to the release branch
+      * *WARNING* Build the package in a clean environment. The reason for that is because the package
+        will contain everything that it is present in the folder. If you are storing credentials or
+        other sensible development information in your folder, they will be uploaded too when we send
+        the package to the ppa. A clean environment is the safest way to perform this.
+
+    b. Edit the changelog:
+      * List yourself as the author of this release.
+      * Edit the version number to look like: `27.2~20.04.1-rc1` (`<version>~<ubuntu-release-number>.<revno>-rc<release-candidate-number>`)
```

Comment:
Maybe we should use `ppa1` instead of `rc1` here. This is just a nit, but is the pattern we used in the past

Reply:
I've seen `ppaX` before, but I prefer `rcX` because it more clearly expresses that this build is a release candidate and not a ppa build for some other reason.

I'm open to arguments otherwise though